user/tests/wiring/no_fixture/random.cpp

```diff
@@ -82,7 +82,7 @@ test(RANDOM_09_zero_crossing_range_values_are_within_range) {
     assertRandomRange(-10,10, 1000);
 }
 
-#if defined(PARTICLE_PROTOCOL) && PARTICLE_PROTOCOL == 0
+#if 0  // TODO - this should work on modular platforms too. This was previously targeting the Core only.
```

Comment:
This feature is not implemented in the non-Core protocol and both Gen 2 and Gen 3 platforms feature hardware RNG. Needs to be removed.